lib/Proxy.js

```diff
@@ -173,8 +177,7 @@ define([
 
 		_publishInSequence: function (message) {
 			var session = this._getSession(message.sessionId);
-
-			if (message.sequence <= session.lastSequence) {
+			if (!isFatalErrorFromPreExecutor(message) && message.sequence <= session.lastSequence) {
```

Comment:
Is this to prevent out-of-sequence errors if the client restarts and then sends a fatal error, or…?